toolkit/Makefile

```diff
@@ -52,6 +52,7 @@ REBUILD_DEP_CHAINS              ?= y
 HYDRATED_BUILD                  ?= n
 DELTA_BUILD                     ?= n
 TARGET_ARCH                     ?=
+ALLOW_PREBUILT_REBUILDS         ?= n
```

Comment:
I'd suggest re-naming this a little. In general rebuilding packages is already allowed and completely fine by default. What we're focusing on are the toolchain package, so I'd suggest something closer to `ALLOW_TOOLCHAIN_REBUILDS`.